README.md

```diff
@@ -186,3 +188,17 @@ emg2pose is CC-BY-NC-SA-4.0 licensed, as found in the LICENSE file.
 emg2pose is also licensed subject to the licenses of its code dependencies.
 
 UmeTrack is licensed under Attribution-NonCommercial 4.0 International, as found in the emg2pose/UmeTrack/LICENSE and [GitHub](https://github.com/facebookresearch/UmeTrack/blob/main/LICENSE).
+
+## Citing emg2pose
+
+```
+@misc{salter2024emg2poselargediversebenchmark,
+      title={emg2pose: A Large and Diverse Benchmark for Surface Electromyographic Hand Pose Estimation}, 
+      author={Sasha Salter and Richard Warren and Collin Schlager and Adrian Spurr and Shangchen Han and Rohin Bhasin and Yujun Cai and Peter Walkington and Anuoluwapo Bolarinwa and Robert Wang and Nathan Danielson and Josh Merel and Eftychios Pnevmatikakis and Jesse Marshall},
+      year={2024},
+      eprint={2412.02725},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV},
+      url={https://arxiv.org/abs/2412.02725}, 
+}
```

Comment:
I know this is inconsistent with emg2qwerty, but I think we should use the NeurIPS paper citation rather than the arXiv one. See the third paper here: https://scholar.google.com/scholar?cluster=6124950150122115814&hl=en&as_sdt=0,31. 

i.e.

```
@inproceedings{salteremg2pose,
  title={emg2pose: A Large and Diverse Benchmark for Surface Electromyographic Hand Pose Estimation},
  author={Salter, Sasha and Warren, Richard and Schlager, Collin and Spurr, Adrian and Han, Shangchen and Bhasin, Rohin and Cai, Yujun and Walkington, Peter and Bolarinwa, Anuoluwapo and Wang, Robert and others},
  booktitle={The Thirty-eight Conference on Neural Information Processing Systems Datasets and Benchmarks Track}
}
```

Reply:
@SSaltEng good idea -- updated!